ydb_dbapi/cursors.py

```diff
@@ -188,6 +224,7 @@ def _get_request_settings(self) -> ydb.BaseRequestSettings:
         return settings
 
     @handle_ydb_errors
+    @invalidate_cursor_on_ydb_error
```

Comment:
**question**: This method doesn't use session, it uses pool. So if the error occur here, this decorator will delete session and transaction, but shouldn't. Example:
```python
with connection.begin():
  connection.execute(DML)
  try:
    connection.execute(DDL) <--- exception occur and session invalidated
  except:
    pass
  connection.execute(DML) <--- fail, because session is lost

```